docs/modules/Domain.ts.md

```diff
@@ -56,7 +56,7 @@ export declare const createClass: (
   signature: string,
   methods: ReadonlyArray<Method>,
   staticMethods: ReadonlyArray<Method>,
-  properties: ReadonlyArray<Property>
+  properties: ReadonlyArray<Property>,
```

Comment:
this is due to a different default in Prettier v3, I would try to add `trailingComma: "none"` to `defaultPrettierOptions`:

```ts
const defaultPrettierOptions: Prettier.Options = {
  parser: "markdown",
  semi: false,
  singleQuote: false,
  printWidth: 120,
  trailingComma: "none"
}
```